### DISCUSSION TITLE
Licencing

### DISCUSSION THREAD

**Discussion by zeropolis79** [community/community#36771] [Other Feature Feedback, Questions, & Ideas]
I'm hosting the source code for a semi-commercial game, but the licence I selected is not compatible with the terms in the readme file.

I DON'T want to licence the code for commercial purpose, but just have it there for research.

What option do I need to select? Thanks

**Comment by jfullstackdev** [comment]
check this out 

https://opensource.stackexchange.com/questions/7687/what-non-commercial-license-is-that

but again, GitHub has a disclaimer in terms of licensing, when an issue arises, it's mostly settled legally

**Comment by ossanna16** [comment]
Thank you for asking a good question @zeropolis79, I'm glad you found our community 🙂
